### ISSUE TITLE
ce_ntp module failure when trying to delete non-existing peer

### ISSUE THREAD

**Issue by amoruck** [HuaweiSwitch/CloudEngine-Ansible#62]
Module ce_ntp called with non-existent peer and proposed "absent" state should report "no change", but obsereved behaviour is "module failure" and netconf error message: "The unicast peer does not exist". Steps to reproduce - run:
```
      ce_ntp:
        peer: 1.1.1.1
        state: absent
        provider: "{{ cli }}"
```

**Comment by yuandongx** [comment]
Thanks for your suggestion We are working hard to update the code to fix this issues.

**Comment by yuandongxx** [comment]
This issues was fixed in version 2.8.2.
You can have a try with latest version.
Thanks.
